lib/registry/src/config.rs

```diff
@@ -36,16 +36,32 @@ pub struct Proxy {
     pub url: Option<String>,
 }
 
-#[derive(Deserialize, Serialize, Debug, PartialEq, Eq, Default)]
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
 pub struct UpdateNotifications {
     pub enabled: String,
```

Comment:
I know you didn't change this here, but this should just be a `bool`.

serde-toml handles that just fine with `true` / `false`.

Reply:
Okay, switched to a new key so it doesn't break wapm-cli.